CONTRIBUTORS.md

```diff
@@ -77,4 +77,4 @@ From oldest to newest contributor, we would like to thank:
 - [Austin Morton](https://github.com/apmorton)
 - [Matt Hammerly](https://github.com/mhammerly)
 - [Christian Vonrüti](https://github.com/alshain)
-
```

Comment:
Nitpicking, but can you make sure there is a blank line after your name? The diff shouldn't be removing anything, just adding your name.